docs/fidesops/docs/postman/Fidesops.postman_collection.json

```diff
@@ -10,6 +10,17 @@
 			"item": [
 				{
 					"name": "Get Root Client Token",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.collectionVariables.set(\"client_token\", pm.response.json()[\"access_token\"])"
+								],
```

Comment:
This script will take the access token from the response and update the collection variable. That way we don't need to manually copy and paste.